code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -142,6 +142,49 @@
 		GLOB.pre_setup_antags -= changeling
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//              ELDRITCH CULT               //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/heretics
+	name = "Heretics"
+	antag_flag = ROLE_CULTIST
+	antag_datum = /datum/antagonist/heretic
+	protected_roles = list("Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	restricted_roles = list("AI", "Cyborg")
+	required_candidates = 1
+	weight = 3
+	cost = 15
+	scaling_cost = 10
+	requirements = list(50,45,45,40,35,20,20,10,10,5)
+	antag_cap = list(1,1,1,2,2,2,2,3,3,3)
```

Comment:
I wouldnt allow this to spawn more than 8 at max, the fact other rulesets do is an oversight.